community/kernel/src/test/java/org/neo4j/kernel/configuration/ConfigTest.java

```diff
@@ -206,4 +209,34 @@ public void shouldBeAbleToAgumentConfig() throws Exception
         assertThat( config.get( setting("non-overlapping", STRING, "") ), equalTo( "huzzah" ) );
         assertThat( config.get( setting("unrelated", STRING, "") ), equalTo( "hello" ) );
     }
+
+    @Test
+    public void shouldProvideViewOfGroups() throws Throwable
+    {
+        // Given
+        Config config = new Config( stringMap(
+                "myusers.0.name", "Bob",
+                "myusers.0.age", "81",
+                "myusers.1.name", "Greta",
+                "myusers.1.age", "82" ) );
```

Comment:
Can we add more tests? What about testing "a.b.0.prop" or "0.prop" kind of names?
What happens when you call `view` and trigger grouping when there is no property names requiring grouping?